tsdb/index/tsi1/partition.go

```diff
@@ -237,6 +245,10 @@ func (p *Partition) Open() error {
 	if err := p.buildSeriesSet(); err != nil {
 		return err
 	}
+	p.tracker.SetSeries(p.seriesIDSet.Cardinality())
```

Comment:
Similar to cache.go, I was able to make a 1:1 mapping from the beta code for most of this. See [here](https://github.com/influxdata/influxdb/blob/v2.0.0-beta.8/tsdb/tsi1/partition.go).